packages/cli-kit/src/public/node/api/admin.ts

```diff
@@ -0,0 +1,62 @@
+import {AdminSession} from '../../../session.js'
+import {content, token as outputToken} from '../../../output.js'
+import {Bug, Abort} from '../../../error.js'
+import {graphqlRequest} from '../../../private/common/api/graphql.js'
+import {ClientError, gql, RequestDocument, Variables} from 'graphql-request'
+
+export async function adminRequest<T>(
+  query: RequestDocument,
```

Comment:
What do you think about making `graphql-request` an internal dependency and change this function signature to:

```js
export async function adminGraphQLRequest<T>(
  query: string,
  session: AdminSession,
  variables: object
)
```
Note that I also included `GraphQL` in the name to make it clear that we are sending a GraphQL request.

Reply:
Yeah, that makes sense, no need to expose graphql-types here 👌